astro/src/content/docs/extend/code/lambdas/steam-reconcile.mdx

```diff
@@ -24,7 +24,7 @@ function reconcile(user, registration, userInfo) {
 This lambda must contain a function named `reconcile` that takes three parameters. The parameters that the lambda is passed are:
 
 <ReconcileLambdaUserRegistrationParameters />
-* `userInfo` - the JSON payload returned by the Steam Token Details API. This is read-only.
+* `userInfo` - the JSON payload returned by the Steam Token Details API. This object is read-only.
```

Comment:
```suggestion
* `userInfo` - the JSON payload returned by the Steam Player Summary API. This object is read-only.
```

Reply:
But again, maybe doesn't make sense to make this change--not sure if it is too much implementation detail.